### ISSUE TITLE
mtl files don't make any effect on obj files

### ISSUE THREAD

**Issue by tomascg2309** [viromedia/viro#773]
- [ x ] Review the documentation: https://docs.viromedia.com/
  - [ x ] Search for existing issues: https://github.com/viromedia/viro
  - [ x ] Use the latest ViroReact release: https://docs.viromedia.com/docs/releases

## Environment
Development OS: Windows
Device OS & Version: Windows 10 Home 64-bit
Version: ViroReact 2.17.0 and React Native 0.59.9

## Description
I'm trying to render 3D Objects (.obj files) and using their materials (.mtl files) but the materials are not rendering properly. For example: the 3d model of the **tent** object looks like this:
![ef3dbd8f-9d0c-48da-a69d-cfbfbb9e5a8d](https://user-images.githubusercontent.com/44789618/68163952-9c262b80-ff29-11e9-9c06-8ce9ddc599f4.jpg)
but when it is rendered at the AR Scene, it looks like this:
![WhatsApp Image 2019-11-04 at 17 50 59](https://user-images.githubusercontent.com/44789618/68164923-10fa6500-ff2c-11e9-9d47-667360109704.jpeg)
Also, the **tree** object has green and brown materials, and renders black.

## Reproducible Demo
Let me share my code:
```
'use strict';

import React, { Component } from 'react';

import { StyleSheet } from 'react-native';

import {
  ViroARScene,
  ViroText,
  ViroConstants,
  ViroBox,
  ViroMaterials,
  Viro3DObject,
  ViroAmbientLight,
  ViroSpotLight,
  ViroARPlaneSelector,
  ViroNode,
  ViroAnimations,
  ViroARTrackingTargets,
  ViroARImageMarker,
  ViroSphere,
  ViroQuad
} from 'react-viro';

export default class MainSceneAR extends Component {

  constructor(props) {
    super(props);

    // Set initial state here
    this.state = {
      texture: "orange_mtl",
      animateScene: false,
      email: props.sceneNavigator.viroAppProps.email
    };

    // bind 'this' to functions
    this._onAnchorFound = this._onAnchorFound.bind(this);
    this._onError = this._onError.bind(this);
  }

  render() {
    return (
      <ViroARScene>
        <ViroARImageMarker
          target={"holidayCard"}
          onAnchorFound={this._onAnchorFound}
          pauseUpdates={this.state.pauseUpdates}
        >
          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tree/Tree.obj")}
              resources={[require("../assets/objects/tree/Tree.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[-.0338,0,-.0399]}
              materials={ "green_mtl" }
              animation={{ name:"scaleBig", run: this.state.animateScene }}
            />
          </ViroNode>

          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tree/Tree.obj")}
              resources={[require("../assets/objects/tree/Tree.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[-.0740,0,.0436]}
              materials={ "green_mtl" }
              animation={{ name:"scaleBig", run: this.state.animateScene }}
            />
          </ViroNode>

          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tree/Tree.obj")}
              resources={[require("../assets/objects/tree/Tree.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[-.0767,0,-.0509]}
              materials={ "green_mtl" }
              animation={{ name:"scaleMedium", run: this.state.animateScene }}
            />
          </ViroNode>

          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tree/Tree.obj")}
              resources={[require("../assets/objects/tree/Tree.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[.0774,0,-.0490]}
              materials={ "green_mtl" }
              animation={{ name:"scaleMedium", run: this.state.animateScene }}
            />
          </ViroNode>

          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tree/Tree.obj")}
              resources={[require("../assets/objects/tree/Tree.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[.0433,0,.0531]}
              materials={ "green_mtl" }
              animation={{ name:"scaleMedium", run: this.state.animateScene }}
            />
          </ViroNode>

          <ViroNode>
            <Viro3DObject
              source={require("../assets/objects/tent/Tent.obj")} 
              resources={[require("../assets/objects/tent/Tent.mtl")]}
              lightReceivingBitMask={3}
              type="OBJ"
              scale={[0,0,0]}
              position={[.0379,0,-.0160]}
              rotation={[0,0,0]}
              // materials={ "white_mtl" }
              animation={{ name:"scaleTent", run: this.state.animateScene }}
              onError={this._onError}
            />
          </ViroNode>

          <ViroSpotLight
            innerAngle={5}
            outerAngle={60}
            direction={[1,-1,-1]}
            position={[-1,3,3]}
            color="#ffffff"
            castsShadow={true}
            shadowMapSize={2048}
            shadowNearZ={2}
            shadowFarZ={7}
            shadowOpacity={.2}
            intensity={100}
          />

          <ViroQuad
            rotation={[-90, 0, 0]}
            position={[0, 0, 0]}
            width={.3} height={.3}
            arShadowReceiver={true}
          />
          
        </ViroARImageMarker>
      </ViroARScene>
    );
  }

  _onAnchorFound() {
    this.setState({
      animateScene: true
    })
  }

  _onError(event) {
    console.log(event.nativeEvent.error);
  }
}

ViroARTrackingTargets.createTargets({
  holidayCard: {
    source: require("../assets/anchors/HolidayCard.png"),
    orientation: "Up",
    physicalWidth: 0.18 // real world width in meters
  }
});

ViroMaterials.createMaterials({
  green_mtl: {
    diffuseColor: "#0B6623",
    lightingModel: "PBR"
  },
  white_mtl: {
    diffuseColor: "#FFFFFF",
    lightingModel: "PBR"
  }
});

ViroAnimations.registerAnimations({
  scaleBig: {
    properties:{scaleX:.01, scaleY:.015, scaleZ:.01,},
    duration: 1000, easing: "bounce",
    delay: 1000
  },
  scaleMedium: {
    properties:{scaleX:.008, scaleY:.012, scaleZ:.008,},
    duration: 800, easing: "bounce",
    delay: 1200
  },
  scaleTent: {
    properties:{scaleX:.00025, scaleY:.00025, scaleZ:.00025,rotateY:"-235"},
    duration: 500, easing: "bounce",
    delay: 2500
  },
  scaleSign: {
    properties:{scaleX:.012, scaleY:.012, scaleZ:.012,},
    duration: 1000, easing: "easein",
    delay: 3000
  }
});

module.exports = MainSceneAR;
```
Also, I can provide you a drive url to download the models:
https://drive.google.com/open?id=1lR60-3zZ5AQc7C_s7c9Pi_yWk1i2DHVA

Please, let me know if there is something I can do to help or if you have any answer! Thanks in advance!

**Comment by tomascg2309** [comment]
Update: it only happens when I render objects from an AR Marker, if I render the object without that (like the Hello World Scene), it renders properly

**Comment by tomascg2309** [comment]
Second update: days ago (I was working without taking the color problem as priority) I noticed that **intensity={100}** is a low value for my spotlight at the distance it is. I adjusted these value gradually above 2500 to apreciate correctly the colors and materials. An .obj file works perfectly, just you (anyone who is reading this for an issue) be sure that the .mtl file has well defined all the textures.
